index.html

```diff
@@ -1,16 +1,54 @@
 <!DOCTYPE html>
 <html>
+    
+    <input type="text" id="mySearch" onkeyup="myFunction()" placeholder="Search.." title="Type in a category">
+
+<ul id="myMenu">
+  <li><a href="">Philosophical Issues</a></li>
```

Comment:
You still need to link to something. You still need this part: `philosophical_issues.html`. Please read what I wrote in detail. I did not ask you to remove the entirety of the link .. only the path leading up to the name of the file:
<img width="802" alt="Screen Shot 2021-02-01 at 11 04 45 AM" src="https://user-images.githubusercontent.com/6777404/106505726-8d585c80-647d-11eb-9e54-d1663709a03b.png">